runtime/pom.xml

```diff
@@ -41,7 +41,7 @@
         <artifactId>test-gen-plugin</artifactId>
         <version>${project.version}</version>
         <configuration>
-          <wastToProcess>labels.wast,
+          <orderedWastToProcess>labels.wast,
```

Comment:
The tests are completely not self-contained, and the execution depends on the ordered sequence of tests.
In this way is going to be a little harder to execute something in isolation, but this implementation better reflects the very nature of those tests.